source/decoupled_authentication.rst

```diff
@@ -0,0 +1,26 @@
+.. _decoupled_authentication:
+
+Decoupled Authentication
+=========================
+
+Decoupled authentication is a feature that allows you to authenticate
+the cardholder during the payment process. This is a different approach to
+challenge flow where the authentication is done in one simultaneous flow.
```

Comment:
It is not clear to me from this part what the difference of the two flow types are :shrug: 

- Decoupled authentication: during payment process
- Non-decoupled authentication: during payment? But isn't this also a payment process?

You only write something about the challenge flow, but I suspect that the same goes for a non challenge flow and that a decoupled flow can also be considered a challenge flow :shrug: 

Do we have some documentation where we can get inspired on how to describe decoupled authentication, and how it differentiates from the other flows?

Reply:
You are right! It can be more clear.
I'm going to rewrite it.
e.g.

A transaction can go through different flows:
- Frictionless flow
- Challenge flow
- Decoupled authentication flow
- SPC flow.

a decoupled authentication flow is a asynchronous authentication process.

Reply:
and include some examples.
AReq, ARes and RReq